CHANGELOG.md

```diff
@@ -4,17 +4,45 @@ Changelog for NeoFS Contract
 ## [Unreleased]
 
 ### Added
-- `verify` method for Alphabet contracts (#386)
-- script comparing some of NeoFS contract states (#399)
 
 ### Updated
-- NeoGo dependency to 0.106.0 (#398)
 
 ### Removed
 
 ### Fixed
 
-### Updating from v0.19.0
+## [0.20.0] - 2024-07-30
+
+### Added
+- `verify` method for Alphabet contracts (#386)
+- Contract deployment code (#395)
+- Functions to deal with address records and New* constructors for nns wrapper (#397)
+- Script comparing some of NeoFS contract states (#399)
+- Script to compare main/fs chain deposit state (#400)
+- Binary contracts provided as Go package (#401)
+- Support for NEP-18 addresses in NNS wrapper (#392)
+- Contract-specific constants and some types to RPC bindings (#402)
+- Prefixes to balance contract storage scheme (#406)
+- `admin` to `properties()` result of NNS (#419)
+
+### Updated
```

Comment:
usually we use `Updated` for dep grades, and `Changed` for changes. Here they are mixed. I'd split them, otherwise any commit can be considered as "update"

Reply:
Anything can be a "change" as well, but I had a feeling I'm missing some section and yeah, that's it.